src/pages/AuthPage/Login.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import Header from "../../components/Header";
+import Layout from "../../components/Layout";
```

Comment:
Create index.js and import all the component is one line

Reply:
+1